internal/dialect/dialectquery/spannerpg.go

```diff
@@ -0,0 +1,44 @@
+package dialectquery
```

Comment:
Spanner via PGAdapter only has partial support for `current_schema`, and we can only query it directly via `select current_schema` but not in a where clause. For this reason, I removed the `Exists` to use the fallback method like the `mssql` dialect does.